lib/bundle/remover.rb

```diff
@@ -0,0 +1,20 @@
+# typed: true
+# frozen_string_literal: true
+
+module Bundle
+  module Remover
+    module_function
+
+    def remove(*args, type:, global:, file:)
+      brewfile = Brewfile.read(global:, file:)
+      content = brewfile.input.split("\n")
+      entry_type = type.to_s if type != :none
+      escaped_args = args.map { |arg| Regexp.escape(arg) }
+      content = content.grep_v(/#{entry_type}(\s+|\(\s*)"(#{escaped_args.join("|")})"/)
+                       .join("\n") << "\n"
+      path = Dumper.brewfile_path(global:, file:)
+
+      Dumper.write_file path, content
```

Comment:
Given `add` is just adding on the end: I wonder about making this just write out the file as-is but with the `remove`d lines removed? Thoughts?

Reply:
This is exactly what we're doing here!

Reply:
Whoops! Was confused by `Dumper` usage.